### PR TITLE
Add Data browser tab

### PR DESCRIPTION
**Pull request by tcjiang9**
Add a tab for the Data browser that is gated behind a feature flag

Local testing

### CODE REVIEW

File: src/components/library-common.js [hunk 1]
```diff
@@ -4,9 +4,15 @@ import { h } from 'react-hyperscript-helpers'
 import { TabBar } from 'src/components/tabBars'
 import TopBar from 'src/components/TopBar'
 import * as Nav from 'src/libs/nav'
+import {isAnalysisTabVisible, isDataBrowserVisible} from "src/libs/config";
```
Comment (dmohs) [inline src/components/library-common.js]:
```suggestion
import {isAnalysisTabVisible, isDataBrowserVisible} from 'src/libs/config';
```

File: src/pages/library/DataBrowser.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+export const navPaths = [
+    {
+        name: 'library-browser',
+        path: '/library/browser',
+        component: DataBrowser,
+        title: 'Browse & Explore'
+    }
+]
```
Comment (dmohs) [inline src/pages/library/DataBrowser.js]:
```suggestion
import {div, h} from 'react-hyperscript-helpers';
import FooterWrapper from 'src/components/FooterWrapper';
import {libraryTopMatter} from 'src/components/library-common';


const DataBrowser = () => {
  return h(FooterWrapper, { alwaysShow: true }, [libraryTopMatter('browse & explore')])
}

export const navPaths = [{
  name: 'library-browser',
  path: '/library/browser',
  component: DataBrowser,
  title: 'Browse & Explore'
}]
```

File: src/libs/routes.js [hunk 3]
```diff
@@ -61,6 +62,7 @@ const routes = _.flatten([
   TermsOfService.navPaths,
   Code.navPaths,
   DataExplorer.navPaths,
+  DataBrowser.navPaths,
```
Comment (dmohs) [inline src/libs/routes.js:64]: Alphabetize.

File: src/pages/library/DataBrowser.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+export const navPaths = [
+  {
+    name: 'library-browser',
+    path: '/library/browser',
+    component: DataBrowser,
+    title: 'Browse & Explore'
+  }
+]
```
Comment (dmohs) [inline src/pages/library/DataBrowser.js]:
```suggestion
export const navPaths = [{
  name: 'library-browser',
  path: '/library/browser',
  component: DataBrowser,
  title: 'Browse & Explore'
}]
```

File: src/libs/config.js [hunk 5]
```diff
@@ -9,7 +9,8 @@ export const getConfig = () => {
 }
 
 export const isAnvil = () => (window.location.hostname === 'anvil.terra.bio') || getConfig().isAnvil
-export const isAnalysisTabVisible = () => false || getConfig().isAnalysisTabVisible
+export const isAnalysisTabVisible = () => getConfig().isAnalysisTabVisible
```
Comment (dmohs) [inline src/libs/config.js:12]: This doesn't seem to be used. Is this change necessary?

File: src/components/library-common.js [hunk 6]
```diff
@@ -3,10 +3,16 @@ import { Fragment } from 'react'
 import { h } from 'react-hyperscript-helpers'
 import { TabBar } from 'src/components/tabBars'
 import TopBar from 'src/components/TopBar'
+import { isAnalysisTabVisible, isDataBrowserVisible } from 'src/libs/config'
```
Comment (dmohs) [inline src/components/library-common.js]:
```suggestion
import { isDataBrowserVisible } from 'src/libs/config'
```

**Review by dmohs — APPROVED** [review]
Approved with requested changes. Re-request review if PR changes substantially.

File: src/libs/config.js (hunk 5 above)
Comment (tcjiang9) [reply src/libs/config.js:12]: I see it used in a few places, like workspacecontainer.js

**Review by zarsky-broad — APPROVED** [review]
